src/object.js

```diff
@@ -781,14 +789,14 @@ module.exports = (common) => {
           })
       })
 
-      it('object.get', (done) => {
+      it('object.get', () => {
         const testObj = {
           Data: new Buffer('get test object'),
           Links: []
         }
 
-        ipfs.object.put(testObj).then((node1) => {
-          ipfs.object.get(node1.multihash).then((node2) => {
+        return ipfs.object.put(testObj).then((node1) => {
+          return ipfs.object.get(node1.multihash).then((node2) => {
```

Comment:
@kenshyx all of these tests are written to test the callback API. Check at the bottom the section to test de promises API.

We always test in depth through the Callback API.